src/main/java/org/kairosdb/datastore/cassandra/CassandraModule.java

```diff
@@ -62,12 +63,10 @@ else if (strKey.startsWith(HECTOR_PREFIX))
 	@Override
 	protected void configure()
 	{
-		bind(Datastore.class).to(CassandraDatastore.class).in(Scopes.SINGLETON);
-		bind(CassandraDatastore.class).in(Scopes.SINGLETON);
-		bind(IncreaseMaxBufferSizesJob.class).in(Scopes.SINGLETON);
-		bind(CleanRowKeyCache.class).in(Scopes.SINGLETON);
-		bind(HectorConfiguration.class).in(Scopes.SINGLETON);
-		bind(CassandraConfiguration.class).in(Scopes.SINGLETON);
+		bind(Datastore.class).to(CassandraDatastore.class).in(SINGLETON);
+		bind(CassandraDatastore.class).in(SINGLETON);
+//		bind(HectorConfiguration.class).in(SINGLETON);
```

Comment:
why it's commented out?

Reply:
Because it's using CQL and Hector is not needed anymore. Will remove the code